src/List.test.jsx

```diff
@@ -0,0 +1,44 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import List from './List';
+
+const handleClick = jest.fn();
+const emptyTasks = [];
+const tasks = [
+  { id: 100, title: '숨 쉬기' },
+  { id: 101, title: '물 마시기' },
+];
+
+const ListComponent = (task) => render(
+  <List
+    tasks={task}
+    onClickDelete={handleClick}
+  />,
+);
```

Comment:
```suggestion
const ListComponent = (task) => render((
  <List
    tasks={task}
    onClickDelete={handleClick}
  />
));
```

괄호로 두 번 감싸주면 콤마를 없앨 수 있어요!